SKILL.tsv

```diff
@@ -4756,37 +4756,37 @@ SKILL_20150717_004755	Range attack rate increases by 3 while [Finestra] is activ
 SKILL_20150717_004756	$Decreases the cooldown time of [Long Stride] by 10 seconds.
 SKILL_20150717_004757	{memo X}$Increases the physical damage increase effect of [Warcry] by 1 per attribute level. (This attribute only applies after the enemy's defense is decreased.)
 SKILL_20150717_004758	$Warcry: Add Target
-SKILL_20150717_004759	The skill [Montano] will inflict the target with [Slow] for 10 secs and the duration of [Slow] will increase by 3secs/2secs/1secs based on Attribute level to small/medium/big sized monsters.
+SKILL_20150717_004759	[Montano] will afflict the target with [Slow] for 10 seconds and the duration of [Slow] will increase by 3/2/1 seconds based on attribute level to small/medium/large sized monsters, respectively.
 SKILL_20150717_004760	{memo X}Spreads [Flame] equal to amount of attribute level when killing enemy under [Flame] with [Targe Smash].
 SKILL_20150717_004761	There is 1% chance per attribute level for enemies pushed away with [Rush] to fall under [Stun] for 2 seconds.
 SKILL_20150717_004762	Endow Earth Property to the attacks of the skill [Earth Wave]
-SKILL_20150717_004763	[Slowdown] is left for 4 seconds after [Arrest], Duration of [Slowdown] will increase 0.4 from Lv2 per Attribute level.
-SKILL_20150717_004764	An enemy that was attacked by [Keelhauling] will be in [Bleeding] status for secs. Duration of [Bleeding] will increase by 1 sec per Attribute level, the bleeding damage will be decided based on character's STR.
+SKILL_20150717_004763	[Slowdown] is applied with a 4 second duration after [Arrest] ends. Duration of [Slowdown] will increase by 0.4 seconds from Lv2 per attribute level.
+SKILL_20150717_004764	An enemy hit with [Keelhauling] will be affected by the [Bleeding] status. Duration of [Bleeding] will increase by 1 sec per attribute level, the bleeding damage is based on your STR.
 SKILL_20150717_004765	$Phalanx: Slow
-SKILL_20150717_004766	When an enemy's attack gets blocked while in [Phalanx] position, the enemy will fall into Slowdown. Duration of Slowdown will increase by 4 secs per Attribute level.
+SKILL_20150717_004766	When an enemy's attack gets blocked while in [Phalanx] position, the enemy will fall into [Slowdown]. Duration of Slowdown will increase by 4 seconds per attribute level.
```

Comment:
When an enemy's attack gets blocked while in [Phalanx] position, the enemy will fall into [Slowdown]. Duration of Slowdown will increase by 4 seconds per attribute level.

to

Inflicts the Slow status effect when an enemy's attack is blocked while in the [Phalanx] formation. Increases the duration of Slow by 4 seconds per attribute level.